PennMobile/General/UserDefaults + Helpers.swift

```diff
@@ -508,3 +508,34 @@ extension UserDefaults {
         removeObject(forKey: UserDefaultsKeys.notificationPreferences.rawValue)
     }
 }
+// MARK: - Two Factor Enabled flag
+extension UserDefaults {
+    func setTwoFactorEnabled(to newValue: Bool) {
+        UserDefaults.standard.set(newValue, forKey: "TOTPEnabled")
+        if newValue {
+            setTwoFactorEnabledDate(Date())
+        } else {
+            setTwoFactorEnabledAgain(to: false)
+        }
+    }
+    
+    func getTwoFactorEnabled() -> Bool {
+        return UserDefaults.standard.bool(forKey: "TOTPEnabled")
```

Comment:
Let's place this key inside UserDefaultsKeys. The main reason is that this will cause the key to be removed on log out. We wouldn't want to ask a logged out user to re-enabled 2FA anyways. In addition, from a code quality perspective, it's almost always a good idea to put repeated strings in a variable name to avoid making mistakes in the future.

Reply:
Done!